ffda-domain-director/luasrc/usr/bin/ffda-switch-domain

```diff
@@ -40,6 +40,14 @@ function switch_domain()
 
   uci:set("gluon", "core", "domain", target_domain)
   uci:commit("gluon")
+  print("Reconfigure wires mesh to use VXLAN")
+  if uci:get("network", "mesh_lan", "legacy") ~= nil then
+    uci:set("network", "mesh_lan", "legacy", '0')
+  end
+  if uci:get("network", "mesh_wan", "legacy") ~= nil then
+    uci:set("network", "mesh_wan", "legacy", '0')
+  end
+  uci:commit("network")
```

Comment:
This change is probably unnecessary now since the domain decides whether to use VXLAN or not.